.gitignore

```diff
@@ -37,6 +37,7 @@ local.properties
 android.iml
 .settings/
 bin/
+android/.settings/
```

Comment:
It is already there just below these lines, the reason why it is still adding your changes is because the files were not deleted from the repo.

You have to delete them (send a PR with the files removed) so that git doesn't track them anymore